tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -275,3 +279,66 @@ TEST(RDFHelpers, SaveGraphToFile)
 
    gSystem->Unlink(outFileName);
 }
+
+TEST(RunGraphs, RunGraphs)
+{
+#ifdef R__USE_IMT
+   ROOT::EnableImplicitMT();
+#endif // R__USE_IMT
+
+   ROOT::RDataFrame df1(3);
+   auto df1a = df1.Define("x", "float(rdfentry_)");
```

Comment:
here and below, prefer not to jit in unit tests, unless you are explicitly testing jitting. integrated over all RDF tests, it makes it a bit faster for devs (mostly me) to run them all many times.

One test case could explicitly test `RunGraphs` + jitting.

```suggestion
   auto df1a = df1.Define("x", [](ULong64_t entry) { return float(rdfentry_); }, {"rdfentry_"});
```

Reply:
Done!

Reply:
not done here -- if this is the one test that should test jitting, can you call it `RunGraphsJitting` or something like that? so I/we don't remove the jitting in a few months by mistake